ios/22/_sentry_arm64.crash

```diff
@@ -0,0 +1,35 @@
+OS Version: iOS 10.2.1 (14D27)
+Report Version: 104
+
+Exception Type: EXC_BREAKPOINT
+Crashed Thread: 0
+
+Application Specific Information:
+Exception 6, Code 771328, Subcode 8
+
+Thread 0 name:
+Thread 0 Crashed:
+0   CrashLibiOS                     0x1000bc500         @objc CRLCrashSwift.crash() -> ()
+1   CrashLibiOS                     0x1000bc500         CRLCrashSwift.crash() -> () (CRLCrashSwift.swift:36)
```

Comment:
I am not sure what you are doing, but there should be only one single frame. This is what the iOS generated crash report looks like, not sure how you are referring to Xcode otherwise. Maybe your crash reporting library is reporting too many frames?

```
0   CrashLibiOS    0x00000001000f052c @objc CRLCrashSwift.crash() -> () (CRLCrashSwift.swift:0)
1   CrashProbeiOS  0x000000010003b840 -[CRLDetailViewController doCrash] (CRLDetailViewController.m:58)
```